docs/concepts/solutions.rst

```diff
@@ -28,43 +34,47 @@ SampleSets
 ----------
 
 Ocean uses the :doc:`dimod <oceandocs:docs_dimod/sdk_index>` :class:`~.SampleSet`
-class to hold samples and some additional information (e.g., timing information from some 
-samplers). 
+class to hold samples and some additional information (e.g., timing and 
+:term:`minor-embedding` information from some samplers). 
 
-For the simple example three-variable "triangular" BQM,
+As a simple example, this three-variable BQM,
 
 .. math::
 
     E(\bf{s}) = - s_0 s_1 - s_0 s_2 + s_1 s_2
     \qquad\qquad s_i\in\{-1,+1\}
 
-might be solved directly on a D-Wave 2000Q system by sampling 1000 times as follows, where the 
-:class:`~.EmbeddingComposite` :term:`composite` maps the symbolic BQM to 
-qubits on the quantum processor, which is called by the 
+might be solved directly on a D-Wave quantum computer by sampling 1000 times. 
+Here, the :class:`~.EmbeddingComposite` :term:`composite` maps the symbolic BQM 
+to qubits on the quantum processor, which is called by the 
 :class:`~.DWaveSampler` sampler:
 
->>> bqm = dimod.BQM({}, {('s0', 's1'): -1, ('s0', 's2'): -1, ('s1', 's2'): 1}, 0, dimod.Vartype.SPIN)  
->>> sampler = EmbeddingComposite(DWaveSampler(solver={'qpu': True}))     
+>>> import dimod 
+>>> from dwave.system import DWaveSampler, EmbeddingComposite
+...
+>>> bqm = dimod.BinaryQuadraticModel(
+...           {}, 
+...           {('s0', 's1'): -1, ('s0', 's2'): -1, ('s1', 's2'): 1}, 
+...           0, 
+...           dimod.Vartype.SPIN)  
```

Comment:
Why keep it simple for this concepts page, like:
```suggestion
>>> bqm = dimod.BQM.from_ising({}, {('s0', 's1'): -1, ('s0', 's2'): -1, ('s1', 's2'): 1})
```

or even :slightly_smiling_face: 
```suggestion
>>> bqm = dimod.BQM.from_ising({}, {'ab': -1, 'bc': 1, 'ca': -1})
```

Reply:
I think we defined that in our examples we use the full ``dimod.BinaryQuadraticModel`` and not the short form ``dimod.BQM`` and use the ``('a', 'b'): 1`` form and not the short ``'ab': 1``. But I like the ``.from_ising``, and will switch to that if you edit one of the above

Reply:
Like this?
```suggestion
>>> bqm = dimod.BinaryQuadraticModel.from_ising({}, {('s0', 's1'): -1, ('s0', 's2'): -1, ('s1', 's2'): 1})
```

The line is too long, and overall it's too verbose. IMO, intro examples should be short and sweet.

Reply:
But sure, this is not the place to deviate from the rest of our docs if we are consistent! Just a food for thought. :slightly_smiling_face:

Reply:
I've shown the `{'ab': -1, 'bc': 1, 'ca': -1}` format before and immediately gotten the question "how do you encode integers?". So I think we should be explicit and use the clearest and most adaptable Python possible.

Reply:
Is it worth revisiting the ``dimod.BinaryQuadraticModel`` versus ``dimod.BQM`` guidance? I think we've cultivated the ``bqm`` pretty thoroughly by now

Reply:
I personally prefer the more explicit long form for docs. I don't think `BQM` is a common enough acronym (yet). Though of course, by that logic, we should be using `.from_quadratic_unconstrained_binary_optimization()`  rather than `.from_qubo()` so...

Reply:
If we want to do short and sweet, IMO we should just go straight to symbolic
```python
a, b, c = dimod.Binaries(['a', 'b', 'c'])
model = -1*a*b + b*c - c*a
```

Reply:
Yes, that's nice. Although it adds another layer to unwrap, at least until we switch to symbolic models all over.

Reply:
Thanks, that's very nice.

One interesting thing I notice is the signs for zero values for a BQM instantiated this way:

```
>>> s0, s1, s2 = dimod.Spins(['s0', 's1', 's2'])
>>> bqm = s1*s2 - s0*s1 - s0*s2
>>> bqm.linear
{'s1': -0.0, 's2': -0.0, 's0': -0.0}
```

We don't get those minus signs when we do ``dimod.BQM({}, {('s0', 's1'): -1, ('s0', 's2'): -1, ('s1', 's2'): 1}, 'SPIN').

Reply:
It's just an artifact of the math, `-0.0 == 0.0` :shrug: